tests/supervisors/test_multiprocess.py

```diff
@@ -75,6 +76,7 @@ def test_multiprocess_run() -> None:
     supervisor.join_all()
 
 
+@pytest.mark.flaky(reruns=3, reruns_delay=2)
```

Comment:
@Kludex For unstable test cases, how about add a retry?

Reply:
No. We should make them stable.

Reply:
This feels a bit difficult, because the service is started with a thread. And the run method involves multiple waits.

https://github.com/encode/uvicorn/blob/master/uvicorn/supervisors/multiprocess.py#L151-L153

Switching execution between threads, as well as killed processes, all affect test stability and assertions.